### PR TITLE
Chapter 1/2 Homework

### PR DESCRIPTION
**Pull request by davinsoman**
While bulky to read, it does satisfy each requirement in the homework. I hope I will be satisfying the pull request as per instructions.

### CODE REVIEW

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 10
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4836333&bid=16062612)

File: Src/BootCamp.Chapter/Program.cs [hunk 1]
```diff
@@ -6,7 +6,41 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string name = "Hancock";
+            string surname = "Johnson";
+            int age = 20;
+            float weight = 81;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'weight'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4836333)

File: Src/BootCamp.Chapter/Program.cs [hunk 2]
```diff
@@ -6,7 +6,41 @@ class Program
...
+            float heightcm = 180;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'heightcm'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4836333)

File: Src/BootCamp.Chapter/Program.cs [hunk 3]
```diff
@@ -6,7 +6,41 @@ class Program
...
+            Console.WriteLine("Age: " + age);
+            Console.WriteLine("Weight: " + weight + "kg");
+            Console.WriteLine("Height: " + heightcm + "cm");
+            Console.WriteLine("BMI: " + BMI);
+
+            Console.WriteLine();
+
+            string name2 = "Debra";
+            string surname2 = "Jones";
+            int age2 = 22;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'age2'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4836333)

File: Src/BootCamp.Chapter/Program.cs [hunk 4]
```diff
@@ -6,7 +6,41 @@ class Program
...
+            float weight2 = 45;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'weight2'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4836333)

File: Src/BootCamp.Chapter/Program.cs [hunk 5]
```diff
@@ -6,7 +6,41 @@ class Program
...
+            float heightcm2 = 165;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'heightcm2'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4836333)

File: Src/BootCamp.Chapter/Program.cs [hunk 6]
```diff
@@ -6,7 +6,41 @@ class Program
...
+            Console.WriteLine("Surname: " + surname);
+            Console.WriteLine("Age: " + age);
+            Console.WriteLine("Weight: " + weight + "kg");
+            Console.WriteLine("Height: " + heightcm + "cm");
+            Console.WriteLine("BMI: " + BMI);
+
+            Console.WriteLine();
+
+            string name2 = "Debra";
+            string surname2 = "Jones";
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'surname2'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4836333)

File: Src/BootCamp.Chapter/Program.cs [hunk 7]
```diff
@@ -6,7 +6,41 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string name = "Hancock";
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'name'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4836333)

File: Src/BootCamp.Chapter/Program.cs [hunk 8]
```diff
@@ -6,7 +6,41 @@ class Program
...
+            string surname = "Johnson";
+            int age = 20;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'age'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4836333)

File: Src/BootCamp.Chapter/Program.cs [hunk 9]
```diff
@@ -6,7 +6,41 @@ class Program
...
+            Console.WriteLine("Name: " + name);
+            Console.WriteLine("Surname: " + surname);
+            Console.WriteLine("Age: " + age);
+            Console.WriteLine("Weight: " + weight + "kg");
+            Console.WriteLine("Height: " + heightcm + "cm");
+            Console.WriteLine("BMI: " + BMI);
+
+            Console.WriteLine();
+
+            string name2 = "Debra";
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'name2'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4836333)

File: Src/BootCamp.Chapter/Program.cs [hunk 10]
```diff
@@ -6,7 +6,41 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string name = "Hancock";
+            string surname = "Johnson";
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'surname'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4836333)

File: Src/BootCamp.Chapter/Program.cs (hunk 7 above)
Comment (diverges) [inline Src/BootCamp.Chapter/Program.cs]: Input should not be hard coded. Use Console.ReadLine(...) to populate these values and then convert them to the appropriate types.

Comment (davinsoman) [reply Src/BootCamp.Chapter/Program.cs]: I misunderstood Step 3, I saw it as making an entire new set of values. The readline was there to bypass a issue that made my code close when I ran it. Figured out what was the issue so Ill adjust the code to allow user input.

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4836333&bid=16062612)

File: Src/BootCamp.Chapter/Program.cs [hunk 11]
```diff
@@ -6,7 +6,45 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            const string name = "Hancock";
+            const string surname = "Johnson";
+            const int age = 20;
+            const float weight = 81;
+            const float heightcm = 180;
+            float heightm = heightcm / 100;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'heightm'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4836333)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4836333&bid=16062612)

File: Src/BootCamp.Chapter/Program.cs [hunk 12]
```diff
@@ -6,7 +6,45 @@ class Program
...
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            const string name = "Hancock";
+            const string surname = "Johnson";
+            const int age = 20;
+            const float weight = 81;
+            const float heightcm = 180;
+            const float heightm = heightcm / 100;
+            float heightsq = heightm * heightm;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'heightsq'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4836333)

File: Src/BootCamp.Chapter/Program.cs (hunk 7 above)
Comment (Alumniminium) [reply Src/BootCamp.Chapter/Program.cs]: Any update on this? Not approving until you read user input into your variables.

Comment (davinsoman) [reply Src/BootCamp.Chapter/Program.cs]: I believe I did it correctly

**Comment by davinsoman** [comment]
I believe I have made all the necessary changes

**Pull request closed without merging**